understanding/22/dragging-movements.html

```diff
@@ -35,24 +35,19 @@ <h2>Intent of Dragging Movements</h2>
 
         <p>Some people cannot perform dragging motions in a precise manner. Others use a specialized or adapted input device such as a head pointer, eye-gaze system, or speech-controlled mouse emulator, which makes dragging cumbersome, error-prone, or outright impossible.</p>
             
-        <p>When an interface implements functionality that uses dragging motions, some users can tap or click, but not accurately maintain contact whilst performing a dragging motion. An alternative method must be provided so that users with mobility impairments that use a pointer (mouse, pen, or touch contact) can use the functionality.</p>
+        <p>When an interface implements functionality that uses dragging movements, some users can tap or click, but not accurately maintain contact whilst performing the movement. An alternative method must be provided so that users with mobility impairments who use a pointer (mouse, pen, or touch contact) can use the functionality.</p>
```

Comment:
The sentence, to me, seems strangely back-to-front. I know this is already in the original, but personally I'd suggest taking the opportunity to turn it around.

```
Users with mobility impairments who use a pointer (mouse, pen, or touch contact) may not be able to accurately maintain contact (such as keeping the mouse button pressed while moving the mouse appropriately). If an interface implements functionality that uses dragging movement, an alternative method must be provided so that these users can operate the content by just using taps or clicks.